Context/ContextSortAttribute.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Markup\NeedleBundle\Context;
+
+use Markup\NeedleBundle\Attribute\AttributeProviderInterface;
+use Markup\NeedleBundle\Config\ContextConfigurationInterface;
+use Markup\NeedleBundle\Sort\RelevanceSort;
+use Markup\NeedleBundle\Sort\Sort;
+
+/**
+ * A context that uses a configuration.
+ */
+class ContextSortAttribute
```

Comment:
Yes actually that's a truly awful name I've used, will fix 

Sent from my iPhone


> On 5 Nov 2015, at 12:55, Douglas Greenshields <notifications@github.com> wrote:
> 
> In Context/ContextSortAttribute.php:
> 
> > @@ -0,0 +1,43 @@
> > +<?php
> > +
> > +namespace Markup\NeedleBundle\Context;
> > +
> > +use Markup\NeedleBundle\Attribute\AttributeProviderInterface;
> > +use Markup\NeedleBundle\Config\ContextConfigurationInterface;
> > +use Markup\NeedleBundle\Sort\RelevanceSort;
> > +use Markup\NeedleBundle\Sort\Sort;
> > +
> > +/**
> > + * A context that uses a configuration.
> > + */
> > +class ContextSortAttribute
> Couldn't this be named better? It's not an attribute - more of a factory function.
> 
> —
> Reply to this email directly or view it on GitHub.
>